File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.c
bool checkIfPangram(char* sentence) {
    int count[26]={0};
    for(int i=0;sentence[i]!='\0';i++)
    {
        int ch=sentence[i];
        if(isalpha(ch))
        {
            ch=tolower(ch);
            count[ch-'a']=1;
        }
    }
    for(int i=0;i<26;i++)
    {
        if(count[i]==0)
        {
            return false;
        }
    }
    return true;

    
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        HashSet<Character> hash=new HashSet<>();
        for(char c:sentence.toCharArray())
        {
            hash.add(c);
        }
        if(hash.size()==26)
        {
            return true;
        }
        return false;
        
    }
}